pkgs/top-level/all-packages.nix

```diff
@@ -7788,12 +7788,24 @@ with pkgs;
 
   godot = callPackage ../development/tools/godot/3 { };
 
-  godot-export-templates = callPackage ../development/tools/godot/3/export-templates.nix { };
+  godot-export-template = callPackage ../development/tools/godot/3/export-template.nix { };
 
   godot-headless = callPackage ../development/tools/godot/3/headless.nix { };
 
+  godot-debug-server = callPackage ../development/tools/godot/3/debug-server.nix { };
+
   godot-server = callPackage ../development/tools/godot/3/server.nix { };
 
+  godot-mono = callPackage ../development/tools/godot/3/mono {};
+
+  godot-mono-export-template = callPackage ../development/tools/godot/3/mono/export-template.nix { };
+
+  godot-mono-headless = callPackage ../development/tools/godot/3/mono/headless.nix { };
+
+  godot-mono-debug-server = callPackage ../development/tools/godot/3/mono/debug-server.nix { };
+
+  godot-mono-server = callPackage ../development/tools/godot/3/mono/server.nix { };
```

Comment:
Lets move that all under godot-mono or similar to not have to many top-level entries. Maybe godoPackages? Not sure

Reply:
I just followed the pattern that was already there; there was a top level entry for each godot derivation.  I'd rather not change that, because I'm not sure what the implications would be.